src/vs/workbench/contrib/terminal/electron-browser/terminalService.ts

```diff
@@ -131,9 +154,12 @@ export class TerminalService extends BrowserTerminalService implements ITerminal
 			useWSLexe = true;
 		}
 
+		const Registry = await import('vscode-windows-registry');
```

Comment:
Let's move the await into `_getShellPathFromRegistry` and make that function async as well